pkg/client/elastic.go

```diff
@@ -37,15 +48,22 @@ func (c ESClient) Current(prefix string) (string, error) {
 	aliases := res.IndicesByAlias(primary)
 	if len(aliases) == 0 {
 		return "", nil
-	} else if len(aliases) > 0 {
+	} else if len(aliases) > 1 {
```

Comment:
I'm curious what has changed to make this count increment... was it just broken before?

Reply:
Yeah, it was broken. The one thing the function was designed to do, it didn't.